medical_compliance/medical_compliance/api/store_utils.py

```diff
@@ -115,7 +111,8 @@ def insert_measurement(store_endpoint_host_uri, user_id, device_id,
 
         "user": user_uri,
         "device": device_uri,
-        "value_info": value_info
+        "value_info": value_info,
+        "ok": True
```

Comment:
@rtud You're totally right, I was blind about this. We'll investigate this when we'll get to create the `contractor` container, as we need to know exactly what the name of the field and it's values should be. Thank you!